`pulumi destroy` crashes on windows

I have a simple stack with a single bucket in it.  I was able to `pulumi update` on windows without a problem, but I'm now running into a problem trying to destroy it:

```
C:\bucket>pulumi destroy
This will permanently destroy all resources in the 'windows-testing' stack!
Please confirm that this is what you'd like to do by typing ("windows-testing"): windows-testing
Destroying stack 'windows-testing' in the Pulumi Cloud
Performing changes:

aws:Bucket("ellismg-bucket"): - Deleting
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x79f9a2]

goroutine 50 [running]:
github.com/pulumi/pulumi/pkg/workspace.GetPluginPath(0xdfe170, 0x8, 0xc04224e260, 0x3, 0x0, 0x1, 0xffffffffffffffff, 0xc04224e260, 0x3, 0xee1280, ...)
        C:/Users/appveyor/go/src/github.com/pulumi/pulumi/pkg/workspace/plugins.go:315 +0x3e2
github.com/pulumi/pulumi/pkg/resource/plugin.NewProvider(0xedf0a0, 0xc0420a6240, 0xc042230740, 0xc04224e260, 0x3, 0x0, 0xed3840, 0xed7540, 0xed9180, 0xed1480)
        C:/Users/appveyor/go/src/github.com/pulumi/pulumi/pkg/resource/plugin/provider_plugin.go:37 +0xc4
github.com/pulumi/pulumi/pkg/resource/plugin.(*defaultHost).Provider.func1(0x40418b, 0xc0420303c0, 0xc042007fa8, 0xed3401)
        C:/Users/appveyor/go/src/github.com/pulumi/pulumi/pkg/resource/plugin/host.go:186 +0x486
github.com/pulumi/pulumi/pkg/resource/plugin.(*defaultHost).loadPlugin.func1(0xc0420303c0, 0xc042007fa8)
        C:/Users/appveyor/go/src/github.com/pulumi/pulumi/pkg/resource/plugin/host.go:125 +0x39
github.com/pulumi/pulumi/pkg/resource/plugin.NewDefaultHost.func1(0xc0420a6240)
        C:/Users/appveyor/go/src/github.com/pulumi/pulumi/pkg/resource/plugin/host.go:72 +0x5c
created by github.com/pulumi/pulumi/pkg/resource/plugin.NewDefaultHost
        C:/Users/appveyor/go/src/github.com/pulumi/pulumi/pkg/resource/plugin/host.go:70 +0x15f
```

This is almost certainly related to how we try to load plugins during a destroy operation, which follows a different path than creating resources (because we need destroy to work when source code is not present).

--- a/pkg/workspace/plugins.go
+++ b/pkg/workspace/plugins.go
@@ -312,7 +312,7 @@ func GetPluginPath(kind PluginKind, name string, version *semver.Version) (strin
 			if m != nil {
 				match = m
 				glog.V(6).Infof("GetPluginPath(%s, %s, %s): found candidate (#%s)",
-					kind, name, *version, match.Version)
+					kind, name, version, match.Version)
 			}
 		}
 	}

